src/crimson/os/seastore/cache.h

```diff
@@ -458,9 +458,8 @@ class Cache {
       SUBTRACET(seastore_cache, "{} {}~{} is absent on t, query cache ...",
                 t, T::TYPE, offset, length);
       const auto t_src = t.get_src();
-      auto f = [&t, this](CachedExtent &ext) {
+      auto f = [&t](CachedExtent &ext) {
 	t.add_to_read_set(CachedExtentRef(&ext));
-	touch_extent(ext);
```

Comment:
if the ext is present in cache, `get_extent<T>()` won't invoke `touch_extent()`, need to fix that.

Reply:
oops, my mistakes. But then it seems that the GC transaction judgement should be added to `touch_extent()` instead of `add_extent()` because the directly call of `touch_extent()` is inevitable.

Reply:
Reasonable to me.